docker-compose.yml

```diff
@@ -54,10 +55,32 @@ services:
         condition: service_started
       s3:
         condition: service_started
+      worker:
+        condition: service_started
     networks:
       - clam-net
       - s3-net
 
+  worker:
+    platform: linux/amd64
+    build:
+      context: .
+      args:
+        POETRY_ARGS: "--no-root --no-ansi"
+    entrypoint: [ "/usr/local/bin/celery", "-A", "request_a_govuk_domain" ]
+    command: [ "worker", "-B", "-l", "INFO" ]
+    environment:
+      <<: *common-application-variables
+      CELERY_BEAT_SCHEDULE_FILENAME: /var/run/request_a_govuk_domain/celerybeat-schedule
+      NOTIFY_EMAIL_FAILURE_RECIPIENT: prakash.jha@digital.cabinet-office.gov.uk
+      ENVIRONMENT: dev
```

Comment:
Can we change the environment to local please? This will avoid conflicting with actual dev environment.

Reply:
I have removed the env variable now, as I don't think I need it